scripts/globals/nyzul.lua

```diff
@@ -753,16 +753,19 @@ xi.nyzul.handleRunicKey = function(mob)
         for _, entity in pairs(chars) do
             -- Does players Runic Disk have data saved to a floor of entering or higher
             if
-                entity:getVar("NyzulFloorProgress") + 1 >= startFloor and
+                entity:getVar("NyzulFloorProgress") > startFloor and
```

Comment:
Are we sure a 0 index isn't used for something else here?

Reply:
Good catch. Reverted it.